data/locale/sv.ts

```diff
@@ -1454,7 +1454,7 @@ Right clicking will bring up a context menu where you can change the order in wh
     <name>EnvelopeAndLfoView</name>
     <message>
         <source>DEL</source>
-        <translation>RADERA</translation>
```

Comment:
:clown_face:

Reply:
This is not only a horribly wrong translation but it fundamentally borks the envelope knob layout. People are busy using the Transifex online interface and don't double check against a compiled real version of lmms.